config.js

```diff
@@ -39,6 +39,11 @@ if (!service) {
 }
 
 process.env.PORT = process.env.PORT  || service.portno
+let redisURL = process.env.REDIS_URL
+const redisPass = process.env.REDIS_PASSWORD
+if (redisPass) {
+  redisURL = `redis://user:${redisPass}@${process.env.REDIS_MASTER_SERVICE_HOST_REDIS}:${process.env.REDIS_MASTER_SERVICE_PORT_REDIS}`
```

Comment:
I would suggest we url encode the password if we are building the connection string.  I would prefer if the env variable is the connection string though.

Reply:
i agree, i just haven't found a way around this currently. does not seem to connect when using 
```
redis://user:PASS@redis:6379
```